pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -126,7 +127,8 @@ let
 
     pos = builtins.unsafeGetAttrPos "version" args;
 
-    inherit patches;
+    patches = patches
+      ++ (lib.optional isCrossBitness ./add-qemu-wrapper-support.patch);
```

Comment:
I’m not really a fan of conditional patches. Can we apply the patch unconditionally (and replace `MAYBE_WRAPPER` with an empty string if we are not using an emulator)?

Reply:
Ok.. in my eyes, it makes it a bit more complex, but ok. There is also an option to just always use a wrapper because, for the native build, it will work as well.